eth/api.go

```diff
@@ -1,3 +1,4 @@
+// +build !mobile
```

Comment:
what's in here, why is this removeD?

Reply:
api implementation. in particular some methods of privatedebugapi and others